ez/ez.go

```diff
@@ -69,6 +76,23 @@ func WithOnWatchedError(cb dials.WatchedErrorHandler) Option {
 	return func(d *dialsOptions) { d.onWatchedError = cb }
 }
 
+// WithFileKeyNaming provides a method for manipulating the casing of the keys
+// in the configuration file.  The decoder argument indicates how to interpret
+// the `dials` tag's formatting.  If the `dials` tag is unspecified, the struct
+// field.  The encoder argument should indicate the format that dials should
```

Comment:
I think this sentence is incomplete.

Reply:
Indeed -- I'll chalk that up to Friday burn out?